packages/graphql-client/src/graphql/user-fileds.fragment.js

```diff
@@ -0,0 +1,8 @@
+import gql from 'graphql-tag';
+
+export const defaultUserFieldsFragment = gql`
+  fragment UserFields on User {
```

Comment:
This is not in the graphql scheme but can be extended from the using code, check out scheme-generator